src/components/message.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import '../styles/components/message.scss';
+import WarningIcon from '../svg/warning.svg';
+import CloseIcon from '../svg/times.svg';
+
+const MessageLevel = {
+  warning: 'warning',
+  failure: 'failure',
+  success: 'success',
+  info: 'info',
+};
+
+const iconSize = '1.125em';
+
+const Message = ({ children, level, onDismiss }) => (
+  <section className={`alert alert--${level}`}>
+    <section className="alert__content">
```

Comment:
Why a section directly inside a section and not a div? Not sure about the semantic meaning here. I'm perfectly happy with the outer section though.